### ISSUE TITLE
"Too many requests from this IP" when working on localhost

### ISSUE THREAD

**Issue by yyunikov** [mudassir0909/stackoverflow-card#6]
When I do requests from my local machine, sometimes I get such type of throttle violations:
<code>
{
"error_id": 502,
"error_message": "too many requests from this IP, more requests available in 17524 seconds",
"error_name": "throttle_violation"
}
</code>

Is there any way I can prevent this?
As far as I can see [here](http://meta.stackexchange.com/questions/243773/stack-exchange-api-too-many-requests-from-this-ip), there can be set a <code>backoff</code> for StackOverflow API.

**Comment by mudassir0909** [comment]
Technically, this happens when you make more than 30 requests per second. https://api.stackexchange.com/docs/throttle

Any reason on why you are making more than 30 request / second ?

**Comment by yyunikov** [comment]
No, this is too much. I just have the stackoverflow card on my page and every time I refresh it I think the call is made. But 30 request / second sounds impossible.

**Comment by mudassir0909** [comment]
Can you provide the link to your page if it's public ?

**Comment by yyunikov** [comment]
Yes, here it is
http://yunikov.com/

**Comment by mudassir0909** [comment]
Okay, I just opened the link it's not happening but is it happening on your IP address ?

**Comment by yyunikov** [comment]
Yes, it was happening on my IP address. And it is already gone actually, but I had to wait for about 24 hours to make the API calls again. I'm just thinking that it can be repeated and I don't think that 30 request / second were made.

**Comment by mudassir0909** [comment]
Hmm...interesting, can you wait for like couple of days & see if it happens again ?

**Comment by yyunikov** [comment]
Yes, sure. If I would receive this again, I will let you know.

**Comment by mudassir0909** [comment]
Alright, cool
